example/spec/parallel/sqsReplay/archivedS3MessagesReplaySpec.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+const delay = require('delay');
+
+const { randomString } = require('@cumulus/common/test-utils');
+const { invokeApi } = require('@cumulus/api-client');
+const { sqs } = require('@cumulus/aws-client/services');
+const { receiveSQSMessages, sendSQSMessage } = require('@cumulus/aws-client/SQS');
+const { getS3KeyForArchivedMessage } = require('@cumulus/ingest/sqs');
+const { s3PutObject } = require('@cumulus/aws-client/S3');
+const {
+  createTimestampedTestId,
+  loadConfig,
+} = require('../../helpers/testUtils');
+
+let asyncOperationId;
+let beforeAllFailed = false;
+let config;
+let queueName;
+let queueUrl;
+let stackName;
+let testName;
+
+// The test setup entails creating SQS messages that will be archived in S3
+describe('The replay archived S3 messages API endpoint', () => {
+  const invalidMessage = JSON.stringify({ testdata: randomString() });
```

Comment:
Is this message actually invalid somehow? If not, we should update the variable name

Reply:
Is it indeed not invalid, nice catch. I will update it

Reply:
[`fb46578` (#2373)](https://github.com/nasa/cumulus/pull/2373/commits/fb46578669e6a77074682bc3a76421fcaf580135)